### PR TITLE
Page update

### PR DESCRIPTION
**Pull request by hippogriffic**
@incuna/frontend please review

not using rems, this isn't for a specific project so there's no jira issue. Icon mixins need to be fixed but I'll do that separately

image

blah
- Edit *
  please ignore `stylesheet.sass` it's a github theme file that I converted to sass but did not actually write myself (sorry @maciejpi )

### CODE REVIEW

File: javascripts/main.js [hunk 1]
```diff
@@ -0,0 +1 @@
+console.log('This would be the main JS file.');
```
Comment (maciejpi) [inline javascripts/main.js:1]: you can probably remove this ?

File: params.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+  "note": "Don't delete this file! It's used internally to help with page regeneration."
+}
```
Comment (maciejpi) [inline params.json:6]: no new line at the end

File: sass/_stylesheet.sass [hunk 3]
```diff
@@ -0,0 +1,291 @@
...
+    font-size: 21px
+
+h5
+    font-size: 18px
+
+h6
+    font-size: 16px
+
+p
+    margin: 10px 0 15px 0
```
Comment (maciejpi) [inline sass/_stylesheet.sass:60]: beth and you did that? really? :)

File: sass/_stylesheet.sass [hunk 4]
```diff
@@ -0,0 +1,291 @@
...
+
+footer a
+    color: #F2F2F2
+    text-decoration: underline
+
+em
+    font-style: italic
+
+strong
+    font-weight: bold
```
Comment (maciejpi) [inline sass/_stylesheet.sass:80]: doesn't `strong` have bold font by default?

File: sass/_stylesheet.sass [hunk 5]
```diff
@@ -0,0 +1,291 @@
...
+    text-shadow: none
+    &:hover, &:focus
+        text-decoration: underline
+
+footer a
+    color: #F2F2F2
+    text-decoration: underline
+
+em
+    font-style: italic
```
Comment (maciejpi) [inline sass/_stylesheet.sass:77]: isn't that default?

File: sass/_stylesheet.sass [hunk 6]
```diff
@@ -0,0 +1,291 @@
...
+    vertical-align: middle
+    text-align: center
+    border: none
+
+pre, code
+    width: 100%
+    color: #222
+    background-color: #fff
+    font-family: Monaco, "Bitstream Vera Sans Mono", "Lucida Console", Terminal, monospace
+    font-size: 14px
```
Comment (maciejpi) [inline sass/_stylesheet.sass:103]: you could nest it

File: sass/_stylesheet.sass [hunk 7]
```diff
@@ -0,0 +1,291 @@
...
+    box-shadow: 0 0 10px rgba(0, 0, 0, 0.1)
+
+pre code
+    display: block
+    box-shadow: none
+
+blockquote
+    color: #666
+    margin-bottom: 20px
+    padding: 0 0 0 20px
```
Comment (maciejpi) [inline sass/_stylesheet.sass:124]: and this

File: sass/_stylesheet.sass [hunk 8]
```diff
@@ -0,0 +1,291 @@
...
+    border-bottom-left-radius: 2px
+    border-bottom-right-radius: 2px
+
+.header-wrap
+    background: #212121
+    background: -moz-linear-gradient(top, #373737, #212121)
+    background: -webkit-linear-gradient(top, #373737, #212121)
+    background: -ms-linear-gradient(top, #373737, #212121)
+    background: -o-linear-gradient(top, #373737, #212121)
+    background: linear-gradient(top, #373737, #212121)
```
Comment (maciejpi) [inline sass/_stylesheet.sass:202]: couldn't we use bourbon for that? I can see it's included in this project

File: sass/_stylesheet.sass [hunk 9]
```diff
@@ -0,0 +1,291 @@
...
+    .inner
+        padding: 50px 10px 30px 10px
+
+.project-title
+    margin: 0
+    color: $orange
+    font-size: 42px
+    font-weight: 700
```
Comment (maciejpi) [inline sass/_stylesheet.sass:210]: nest?

File: sass/_stylesheet.sass [hunk 10]
```diff
@@ -0,0 +1,291 @@
...
+    text-shadow: #111 0px 0px 10px
+
+.project-tagline
+    color: #fff
+    font-size: 24px
+    font-weight: 300
```
Comment (maciejpi) [inline sass/_stylesheet.sass:216]: and this

File: sass/_stylesheet.sass [hunk 11]
```diff
@@ -0,0 +1,291 @@
...
+.zip_download-link:hover
+    background: url(../images/sprite_download.png) no-repeat top left
+
+.tar-download-link:hover
+    background: url(../images/sprite_download.png) no-repeat top right
+
+.main-content-wrap
+    background: #eee
+    border-top: 1px solid #111
+    border-bottom: 1px solid #111
```
Comment (maciejpi) [inline sass/_stylesheet.sass:257]: and this

File: sass/_stylesheet.sass [hunk 12]
```diff
@@ -0,0 +1,291 @@
...
+
+.main-content
+    padding-top: 40px
+
+.footer-wrap
+    background: #212121
+
+// Small Device Styles
+@media screen and (max-width: 480px)
```
Comment (maciejpi) [inline sass/_stylesheet.sass:266]: no mobile first? :)

**Comment by maciejpi** [comment]
@hippogriffic finished commenting. a few small comments, nothing major

File: sass/_stylesheet.sass (hunk 3 above)
Comment (hippogriffic) [inline sass/_stylesheet.sass:60]: oh, no, sorry should have said this one is a basic github stylesheet that I converted to sass, I didn't actually write this

Comment (maciejpi) [inline sass/_stylesheet.sass:60]: oh I see. so do you want me to merge it?

**Comment by ninjanails** [comment]
I think it looks good 👍

File: params.json [hunk 13]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "Incuna-transitions",
+  "tagline": "blah",
```
Comment (pandalion) [inline params.json:3]: change blah maybe ?

**Comment by pandalion** [comment]
Looks nice :D

File: params.json (hunk 13 above)
Comment (hippogriffic) [inline params.json:3]: ha whoops

Comment (hippogriffic) [inline params.json:3]: I will fix that in my new branch

**Pull request merged**